web/wp-content/plugins/ad-stack/inc/ads-global-targeting.php

```diff
@@ -32,19 +69,18 @@ function ads_global_targeting_parameters() {
 	$url_part = $unslash_url;
 
 	$g_targeting = array(
-		'property' => '6178',
-		'siteId'   => 'cpt',
-		'pageType' => $page_type,
-		'urlPath'  => $url_part,
-		'keyWords' => get_post_tags(),
-		'category' => $top_level_categories,
-		'topic'    => $sub_categories,
-	);
-	$breakpoints = array(
-		'breakpoint' => get_breakpoints(),
+		'global_targeting' => array(
+			'property' => get_dfp_property(),
```

Comment:
This will be passed to DFP as a targeting param. Is it intentional? Is it the current behavior?

Reply:
only the global targeting will go to dfp. its been take care